data/tooltips.txt

```diff
@@ -75,10 +75,10 @@ tip "cargo scan:"
 	`Lets you scan your target ship's cargo from up to this distance away.`
 
 tip "cargo scan power:"
-	`The range of this cargo scanner is proportional to the square root of this value. The speed of the scanner also decreases in proportion to the square root of this value.`
+	`The range of this cargo scanner is proportional to the square root of this value. At increasing distances, the scanner's speed will naturally decline.`
 
 tip "cargo scan speed:"
-	`The time it takes for this scanner to complete is inversely proportional to this value. Many factors can impact the speed of a scan, like the size of the target ship, its distance to the scanner, or its velocity relative to the scanner.`
+	`The time it takes for this scanner to complete, all things equal, is inversely proportional to this value. Multiple factors can impact the speed of a scan, like the size of the target ship or its distance to the scanner.`
```

Comment:
Definitely clearer than before, although it might be better yet to have a simpler lead off.
```suggestion
	`The higher this value, the faster a scan will complete. The time it takes for this scanner to complete, all things equal, is inversely proportional to this value. Multiple factors can impact the speed of a scan, like the size of the target ship or its distance to the scanner.`
```

Reply:
I think it may be better to rapidly prototype tooltips on discord, so we can quickly settle on something good without the obligatory formality of github. I'm thinkin', like...

`The higher this value, the faster a scan will complete. Multiple factors can impact the speed of a scan, bla bla bla etc` because if it's too long of a tooltip it may be difficult to parse out. (even if the longest tooltip takes less than 5 seconds to read out. humans simply have low tolerance for visually-intimidating text.)